CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 2.8)
 
 project(synfig-studio)
 
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib/modules)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib/modules)
```

Comment:
I suggest to set `CMAKE_LIBRARY_OUTPUT_DIRECTORY ` to `${CMAKE_BINARY_DIR}/lib/`
and inside `synfig-core/src/modules/CMakeLists.txt` set to `${CMAKE_BINARY_DIR}/lib/modules/`